### PR TITLE
fix: stop formatting app property

### PR DESCRIPTION
**Pull request by wa0x6e**
Toward https://github.com/snapshot-labs/workflow/issues/235
Discussion https://discord.com/channels/955773041898573854/1296435871431786587

This PR will accept ~~longer and~~ unformatted `app` property for proposal and vote actions.

### Changes

- ~~`app` chars limit bumped from 24 to 128 chars (to accept potential address)~~
- remove kebabcase formatting

Run the following SQL query to bump the column length:

```sql 
ALTER TABLE
  votes
MODIFY
  app varchar(128) NOT NULL;
```

```sql 
ALTER TABLE
  proposals
MODIFY
  app varchar(128) NOT NULL;
```

### CODE REVIEW

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -208,3 +208,9 @@ export function captureError(e: any, context?: any, ignoredErrorCodes?: number[]
 export async function clearStampCache(type: string, id: string) {
   return fetch(`https://cdn.stamp.fyi/clear/${type}/${type === 'avatar' ? 'eth:' : ''}${id}`);
 }
+
+export function formatApp(app: string | undefined): string {
+  if (!app || app?.length > 128) return '';
```
Comment (ChaituVR) [inline src/helpers/utils.ts:213]: We should update on snapshot.js instead

https://github.com/snapshot-labs/snapshot.js/blob/master/src/schemas/proposal.json#L86

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — APPROVED** [review]
utAck

**Comment by wa0x6e** [comment]
PR reverted to original 24 chars limit for `app`.

Will update the limit again when database will be updated

**Review by bonustrack — APPROVED** [review]

**Comment by ChaituVR** [comment]
Need to revert on snapshot.js as well?

**Comment by wa0x6e** [comment]
> Need to revert on snapshot.js as well?

Correct, see https://github.com/snapshot-labs/snapshot.js/pull/1098

**Pull request merged**